Tests/UI/New UI/BillingForm/Validator/PostCodeValidatorTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import Frames
+
+class PostCodeValidatorTests: XCTestCase {
+
+    func testValidationWhenTextIsEmptyThenShouldBeTrue() {
+        let expectedType = BillingFormCellType.postcode
+        let text = ""
+        let isValid = expectedType.validator.validate(text: text)
+        XCTAssertEqual(isValid, true)
+    }
+    
+    func testValidationWhenTextIsNonEmptyThenShouldBeFalse() {
```

Comment:
There is no test to validate correct functionality, just the no input and the invalid format.

Reply:
please explain more.

Reply:
I looked into the PostCodeValidator now and I can see that it only checks if empty or not, so I suppose I was expecting some type of confirmation the postcode validation works correctly.

I now see that's not the case, but I do need to now ask if creating many validators that do the exact same thing (check value exists) is a good solution. We end up with many identical placeholders, that whilst I can see how they can grow, I also see they are not used and (knowing software development) are probably going to go empty for a long time, potentially forever.

Should we just have a validator `ContentExistsValidator` if we are very keen on following with validators. Then as we find ways to test postcodes, names or other functionality, we replace as and when needed with the correct fit validator for that use case.

Reply:
I will add more validation for some fields like country, phone number, name, etc in the next ticket